LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 the OWASP Core Rule Set contributors
+   Copyright  contributors
```

Comment:
Broken replace? All over the place I guess...

Reply:
But this one here is a bit troubling indeed.

Reply:
fixed